packages/app/src/app/pages/Dashboard/Components/Branch/BranchListItem.tsx

```diff
@@ -29,49 +32,64 @@ export const BranchListItem = ({ branch, onClick }: BranchProps) => {
         backgroundColor: 'transparent',
         color: 'inherit',
         ':hover, :focus, :focus-within': {
-          cursor: 'default',
           backgroundColor: 'list.hoverBackground',
         },
       })}
     >
-      <Grid css={{ width: 'calc(100% - 26px - 8px)' }} columnGap={2}>
-        <Column
-          span={[12, 7, 7]}
-          css={{
-            display: 'block',
-            overflow: 'hidden',
-            paddingBottom: 4,
-            paddingTop: 4,
-          }}
-        >
-          <Stack gap={4} align="center" marginLeft={2}>
-            {contribution ? (
-              <Icon color="#EDFFA5" name="contribution" size={16} />
-            ) : (
-              <Icon name="branch" size={16} />
-            )}
+      <Element
```

Comment:
just adding an anchor inside the list item here, to ensure the entire row is clickable